src/monodroid/jni/monodroid-glue.cc

```diff
@@ -1418,6 +1451,45 @@ MonodroidRuntime::typemap_managed_to_java (MonoReflectionType *type, const uint8
 	return embeddedAssemblies.typemap_managed_to_java (type, mvid);
 }
 
+#if defined (WINDOWS)
+const char*
+MonodroidRuntime::get_my_location ()
+{
+	HMODULE hm = NULL;
+
+	if (GetModuleHandleEx (GET_MODULE_HANDLE_EX_FLAG_FROM_ADDRESS |
```

Comment:
As per elsewhere, we should *avoid* use of the "macro WinAPI functions", and use the "real" functions directly.  Furthermore, we should avoid use of the `A`Functions, and should prefer the `W` functions.

As such:

```cpp
DWORD handle_flags = GET_MODULE_HANDLE_EX_FLAG_FROM_ADDRESS | GET_MODULE_HANDLE_EX_FLAG_UNCHANGED_REFCOUNT;
if (GetModuleHandleExW (handle_flags, (LPCWSTR) &get_xamarin_android_msbuild_path, &hm) == 0) {
    …
}
```